stable/datadog/README.md

```diff
@@ -328,6 +328,10 @@ helm install --name <RELEASE_NAME> \
 | `datadog.systemProbe.enableConntrack`                        | If true, system-probe connects to the netlink/conntrack subsystem to add NAT information to connection data. Ref: http://conntrack-tools.netfilter.org/                             | `true`                                          |
 | `datadog.systemProbe.bpfDebug`                               | If true, system-probe writes debug logs to /sys/kernel/debug/tracing/trace_pipe                                                                                                     | `false`                                         |
 | `datadog.systemProbe.apparmor`                               | Apparmor profile for system-probe                                                                                                                                                   | `unconfined`                                    |
+| `datadog.securityAgent.enabled`                              | Enable security agent                                                                                                                                                               | `false`                                         |
```

Comment:
Should not exist, can be deduced from `.compliance.enabled || .security.enabled` I guess

Reply:
This is following the approach similar to `processAgent` - where the container is enabled and then features are configured for it.